lib/DDGC/Base/Web/App.pm

```diff
@@ -0,0 +1,29 @@
+package DDGC::Base::Web::App;
+
+# ABSTRACT: Base module for Applications (rendering services)
+
+use Import::Into;
+
+use strict;
+use warnings;
+use utf8;
```

Comment:
No, but this gets applied to source which deals with templates / other data so there's potential for utf8 in source.